src/test/java/org/minions/devfund/richard/moviedatabase/MovieTest.java

```diff
@@ -0,0 +1,50 @@
+package org.minions.devfund.richard.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Test Class.
+ */
+public class MovieTest {
+
+    private Movie movie;
+
+    /**
+     * Before.
+     */
+    @Before
+    public void setUp() {
+        movie = new Movie();
+    }
+
+    /**
+     * Test.
+     */
+    @Test
+    public void testAddActorAlreadyExist() {
+        final Actor actor1 = new Actor();
+        final Actor actor2 = new Actor();
+        actor1.setName("Actor1");
+        actor2.setName("Actor2");
+        movie.setActors(actor1);
+        movie.setActors(actor2);
+        movie.setActors(actor1);
+        assertEquals(2, movie.getActors().size());
+        assertEquals("Actor1", movie.getActors().get(0).getName());
+        assertEquals("Actor2", movie.getActors().get(1).getName());
+    }
+
+    /**
+     * Test.
+     */
+    @Test
+    public void testEquals() {
+        assertFalse(movie.equals(null));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this call to "equals"; comparisons against null always return false; consider using '== null' to check for nullity. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2159)